src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -22,50 +22,45 @@
     ]);
     vm.hideDragMe = false;
     vm.activationPromise = activate();
-    var defaultPlanId = 6;
+    var defaultPlanName = 'PLAN_60K';
     var planItems;
+    var planItemsParsedForSlider;
+    vm.langUsed = $translate.use();
 
     function activate() {
       return settings.getPlansAvailable().then(function(response){
-        planItems = response.data;
-        changePlan(defaultPlanId);
+        planItems = response.data.plans;
+        planItemsParsedForSlider = response.data.plans.map(function(plan){
```

Comment:
What do you think about removing `planItemsParsedForSlider` field and apply this logic inside `loadSlider()`?